packages/components/label/src/label.tsx

```diff
@@ -38,6 +38,7 @@ export type TLabelProps = {
   htmlFor?: string;
   // Indicates if the label title should be in bold text
   isBold?: boolean;
+  fontWeight?: 'regular' | 'medium' | 'bold';
```

Comment:
There is no comment for fontWeight prop. Not sure if that's a consistency thing but it looks like all the others have a comment. Not sure if these comments are used in documentation somewhere.

Reply:
I'll add one just to be safe